packages/firebase_analytics/firebase_analytics/lib/src/firebase_analytics.dart

```diff
@@ -125,14 +125,26 @@ class FirebaseAnalytics extends FirebasePluginPlatform {
     );
   }
 
-  /// Sets the applicable end user consent state. 'default' value for 'adStorageConsentGranted' & 'analyticsStorageConsentGranted' is 'true'
+  /// Sets the applicable end user consent state.
+  /// By default, no consent mode values are set. 
+  ///
+  /// Default consents can be set according to the platform: 
+  /// - [iOS][1]
+  /// - [Android][2]
+  ///
+  /// [1]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=ios#default-consent
+  /// [2]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=android#default-consent
   Future<void> setConsent({
     bool? adStorageConsentGranted,
     bool? analyticsStorageConsentGranted,
+    bool? adPersonalizationSignalsConsentGranted,
+    bool? adUserDataConsentGranted,
```

Comment:
We also need to add 3 other consents for web platform here. See: https://firebase.google.com/docs/reference/js/analytics.consentsettings?_gl=1*kcmshc*_up*MQ..*_ga*MTk0NzA2OTc0Ni4xNzEwMjY0MDEx*_ga_CW55HF8NVT*MTcxMDI2NDAxMC4xLjAuMTcxMDI2NDAxMC4wLjAuMA..

Reply:
Done